drivers/rtc/rtc-hym8563.c

```diff
@@ -473,9 +474,15 @@ static irqreturn_t hym8563_irq(int irq, void *dev_id)
 static int hym8563_init_device(struct i2c_client *client)
 {
 	int ret;
+	int retry;
 
 	/* Clear stop flag if present */
-	ret = i2c_smbus_write_byte_data(client, HYM8563_CTL1, 0);
+	for (retry=10; retry>0; retry--) {
+		ret = i2c_smbus_write_byte_data(client, HYM8563_CTL1, 0);
+		if (ret >= 0)
+			break;
+		msleep(10);
+	}
 	if (ret < 0)
 		return ret;
```

Comment:
```suggestion
		return -EPROBE_DEFER;
```
应该是保持原来的函数调用不变，只是告诉系统让他之后重新尝试probe，而不是自己把这里卡死

Reply:
这样是想让PMU上的RTC不会先注册，如果返回-EPROBE_DEFER，会不会被PMU-RTC给先注册了呢。

Reply:
那去改驱动给pmu rtc加个开关的flag啊……从根源解决问题